performance/buildSrc/src/main/kotlin/RunKotlinNativeTask.kt

```diff
@@ -15,14 +15,17 @@ import java.io.File
 
 open class RunKotlinNativeTask @Inject constructor(
         private val curTarget: KotlinTarget
-): DefaultTask() {
+) : DefaultTask() {
 
     var buildType = "RELEASE"
     var workingDir: Any = project.projectDir
     var outputFileName: String? = null
     @Input
-    @Option(option = "filter", description = "filter")
+    @Option(option = "filter", description = "Benchmarks to run (comma-separated)")
     var filter: String = ""
+    @Input
+    @Option(option = "filterRegex", description = "Benchmarks to run, as regexes (comma-separated)")
```

Comment:
May be "Benchmarks to run  described by regular expressions" is better?

Reply:
Done.